src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -173,14 +306,24 @@ int zocl_del_context_kernel(struct drm_zocl_dev *zdev, void *client_hdl, u32 cu_
 	return ret;
 }
 
+/*
+ * Detele an existing context and remove it from the kds.
+ *
+ * @param	zdev:   zocl device structure
+ * @param       client:	KDS client structure
+ * @param       args:	userspace arguments
+ *
+ * @return      0 on success, error core on failure.
+ *
+ */
 static int
 zocl_del_context(struct drm_zocl_dev *zdev, struct kds_client *client,
 		 struct drm_zocl_ctx *args)
 {
 	struct kds_ctx_info info;
```

Comment:
Uninitialized structure
```suggestion
	struct kds_ctx_info info = {0};
```